### PR TITLE
Post request to obc - Airfrop Target 

### PR DESCRIPTION
**Pull request by MatthewHsu1**
Finish GSC route handler that reads in the vector of <AirdropTargets> objects and sends it to the airdrop client.

### CODE REVIEW

File: src/network/gcs_routes.cpp
Comment (Tyler-Lentz) [inline src/network/gcs_routes.cpp:306]: this can be removed and then this pr should be ready to merge (I fixed the tick lock ptr so that it this isn't needed, since the has value call should catch if it is the wrong tick)

File: src/network/gcs_routes.cpp
Comment (Tyler-Lentz) [inline src/network/gcs_routes.cpp:318]: this too

**Review by Tyler-Lentz — COMMENTED** [review]
This is super sweet, thank you, just have a tiny thing from the merges than can be removed

ill probably handle this when its ready to merge, or if you see this before its been merged you can do it if you would like

File: src/network/gcs_routes.cpp
Comment (MatthewHsu1) [reply src/network/gcs_routes.cpp]: Done

File: src/network/gcs_routes.cpp
Comment (MatthewHsu1) [reply src/network/gcs_routes.cpp]: Done

**Review by Tyler-Lentz — APPROVED** [review]

**Comment by Tyler-Lentz** [comment]
holy shit you even went back and fixed the lint :goat:

**Review by Samir-Rashid — APPROVED** [review]

**Pull request merged**
